proposals/0029-cooperative-vector.md

```diff
@@ -749,64 +760,64 @@ explicitly checked for the combinations below.
 #### Usage Example
 
 ```c++
-// Check for CooperativeVector support and query properties for MatrixVectorMulAdd
-D3D12_FEATURE_DATA_D3D12_OPTIONSNN CoopVecSupport = {};
+// Check for matrix vector support and query properties for MatrixVectorMulAdd
+D3D12_FEATURE_DATA_D3D12_OPTIONSNN MatVecSupport = {};
 
-d3d12Device->CheckFeatureSupport(D3D12_FEATURE_D3D12_OPTIONSNN, &CoopVecSupport, 
+d3d12Device->CheckFeatureSupport(D3D12_FEATURE_D3D12_OPTIONSNN, &MatVecSupport, 
                                  sizeof(D3D12_FEATURE_DATA_D3D12_OPTIONSNN));
 
-if (CoopVecSupport.CooperativeVectorTier == D3D12_COOPERATIVE_VECTOR_TIER_1_0) {
+if (MatVecSupport.LinearAlgebraMatrixVectorTier == D3D12_LINEAR_ALGEBRA_MATRIX_VECTOR_TIER_1_0) {
     // PropCounts to be filled by driver implementation
-    D3D12_FEATURE_DATA_COOPERATIVE_VECTOR CoopVecProperties = {0, NULL, 0, NULL, 0, NULL};
+    D3D12_FEATURE_DATA_LINEAR_ALGEBRA_MATRIX_VECTOR MatVecProperties = {0, NULL, 0, NULL, 0, NULL};
 
     // CheckFeatureSupport returns the number of input combinations for inference intrinsic
-    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_COOPERATIVE_VECTOR, &CoopVecSupport, 
-                                     sizeof(D3D12_FEATURE_COOPERATIVE_VECTOR));
+    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_LINEAR_ALGEBRA_MATRIX_VECTOR, &MatVecProperties, 
+                                     sizeof(D3D12_FEATURE_LINEAR_ALGEBRA_MATRIX_VECTOR));
 
     // Use MatrixVectorMulAddPropCount returned from the above
 
     // Use CheckFeatureSupport call to query only MatrixVectorMulAddProperties
-    UINT MatrixVectorMulAddPropCount = CoopVecSupport.MatrixVectorMulAddPropCount;
-    std::vector<D3D12_COOPERATIVE_VECTOR_PROPERTIES_INFERENCE> properties(MatrixVectorMulAddPropCount);
-    CoopVecSupport.pMatrixVectorMulAddProperties = properties.data();
+    UINT MatrixVectorMulAddPropCount = MatVecProperties.MatrixVectorMulAddPropCount;
+    std::vector<D3D12_LINEAR_ALGEBRA_MATRIX_VECTOR_PROPERTIES_INFERENCE> properties(MatrixVectorMulAddPropCount);
+    MatVecProperties.pMatrixVectorMulAddProperties = properties.data();
 
     // CheckFeatureSupport returns the supported input combinations for the inference intrinsic
-    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_COOPERATIVE_VECTOR, &CoopVecSupport, 
-                                    sizeof(D3D12_FEATURE_DATA_COOPERATIVE_VECTOR));
+    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_LINEAR_ALGEBRA_MATRIX_VECTOR, &MatVecProperties, 
+                                    sizeof(D3D12_FEATURE_DATA_LINEAR_ALGEBRA_MATRIX_VECTOR));
                                                                 
     // Use MatrixVectorMulAdd shader with datatype and interpretation
     // combination matching one of those returned.
     
 } else {
-    // Don't use Cooperative Vector
+    // Don't use matrix vector ops
```

Comment:
should this be `Don't use LINEAR_ALGEBRA_MATRIX_VECTOR ops` just to avoid misunderstanding with the native `vector` and `matrix` datatypes.

Reply:
fixed